Formula/ledger.rb

```diff
@@ -57,6 +61,11 @@ def install
     system "./acprep", "opt", "make", "doc", *args
     system "./acprep", "opt", "make", "install", *args
 
+    xy = Language::Python.major_minor_version Formula["python@3.9"].bin/"python3"
+    (lib/"python#{xy}/site-packages/homebrew-ledger.pth").write <<~EOS
+      import site; site.addsitedir('#{libexec}')
+    EOS
```

Comment:
Can we install this directly into the `prefix` and not `libexec`? Having to use a `.pth` file isn't great. See #73875.

Reply:
Ah, okay. I followed the `.pth` example from, e.g., the NumPy formula. I tried to change it to what I think you meant (I'm new to Homebrew Formulas). PTAL.